source/extensionPoints/__init__.py

```diff
@@ -66,22 +70,28 @@ def notifyOnce(self, **kwargs):
 				log.exception(f"Error running handler {handler} for {self}. Exception {e}")
 
 
-FilterValueTypeT = TypeVar("FilterOutputTypeT")
+FilterValueTypeT = TypeVar("FilterValueTypeT")

```

Comment:
should fixing this misnamed TypeVar be done in a separate PR, targeting 2023.1?

Also I think `TypeT` is redundant, `T` is used to indicate type, unless this a "type type"

```suggestion
FilterValueT = TypeVar("FilterValueT")
```